app/views/admins/index.html.erb

```diff
@@ -1,9 +1,26 @@
-<%=@user.first_name%>
+Welcome <%= current_user.first_name %> | <%= link_to 'Facilities', show_facilities_admins_path %>
 
-<% @items.each do |item|%>
-<%=item.name%>
-<%end%>
+<div>
+  <div><h1>Pending Items</h1><%= link_to 'All Items', show_items_admins_path %></div>
+  <% @items.each do |item|%>
+    <div>
+      <%=item.name%> <%= item.date %> <%= item.description %> <%= item.price %> <%= item.status %> <%= item.facility_id %>
+      <%= link_to 'Approve', approve_item_admin_path(item.id), method: :'post' %>
+      <%= link_to 'Edit', edit_item_path(item.id) %>
+      <%= link_to 'Delete', item_path(item.id), method: :'delete' %>
```

Comment:
For all of these links, you don't need to specify `item.id`. (Check out the [examples at the end of this section](https://guides.rubyonrails.org/getting_started.html#updating-articles) of the Rails Guides.)

```suggestion
      <%= link_to 'Delete', item_path(item), method: :'delete' %>
```